src/frontends/tensorflow/src/input_model.cpp

```diff
@@ -394,6 +394,35 @@ ov::frontend::Place::Ptr InputModel::InputModelTFImpl::get_place_by_tensor_name(
     size_t port_idx;
     std::string port_type;
     tensorflow::extract_operation_name_and_port(tensorName, operation_name, port_idx, port_type);
+
+    if (m_saved_model_input_names.get()) {
+        for (const auto& alt_name : *m_saved_model_input_names) {
+            if (alt_name.second == operation_name) {
+                auto pos = alt_name.first.find_first_of(':');
+                if (pos != std::string::npos) {
+                    operation_name = alt_name.first.substr(0, pos);
+                } else {
+                    operation_name = alt_name.first;
+                }
+                break;
+            }
+        }
+    }
+
+    if (m_saved_model_output_names.get()) {
+        for (const auto& alt_name : *m_saved_model_output_names) {
```

Comment:
nitpick: these two for loops are very similar, can be extracted to a function called twice

Reply:
It is place I want to be reviewed by @rkazants. It is needed to fix issue with names. And yes it might be replaced by a function/lambda if it is correct change for names.